core/src/mindustry/game/SectorInfo.java

```diff
@@ -27,6 +27,8 @@
     public ObjectMap<Item, ExportStat> rawProduction = new ObjectMap<>();
     /** Export statistics. */
     public ObjectMap<Item, ExportStat> export = new ObjectMap<>();
+    /** Import statistics. */
+    public ObjectMap<Item, ExportStat> imports = new ObjectMap<>();
```

Comment:
Imports should be calculated on-demand when the import dialog is opened - they aren't state.

Reply:
Ah yeah I just did that so that we don't have to rebuild and gc the object every time. Should it be transient or just local to the method?

Reply:
Local to the method. Allocating a ObjectMap when a very specific button is pressed outside of the main game is inconsequential.